.gitignore

```diff
@@ -2,7 +2,7 @@
 .idea/
 .~c9*
 .vscode/
-nikitas_testingdedug.py
+nikitas_testingdebug.py
```

Comment:
What did you make a testing debug file for? If you were testing the code itself, you should just write a new unit test. You can do normal python debugging inside of pytest